packages/server/test/unit/config_spec.js

```diff
@@ -622,10 +622,22 @@ describe('lib/config', () => {
           return this.expectValidationPasses()
         })
 
+        it('passes if true', function () {
+          this.setup({ videoCompression: true })
+
+          return this.expectValidationPasses()
```

Comment:
This doesn't test that run mode is overriding this to be 32.

Reply:
correct. That currently happens in the system tests with video enabled

Reply:
I think I can add a system-test for this that sets videoCompression to `true` and validates the output in the terminal is 32 CRF on the compression since I don't think we have unit tests for the the run mode file

Reply:
👍🏻 sounds good. Can we set it up to also record so we ensure that the config value passed to the cloud is `true` and not `32`?

Reply:
yeah I can take care of that. I am wondering if the `record_spec` system test is most appropriate to add this?